src/source/source_cache.js

```diff
@@ -392,6 +393,27 @@ class SourceCache extends Evented {
         this._cache.setMaxSize(maxSize);
     }
 
+    handleWrapJump(lng: number) {
+        // Wrapping longitude values can cause a jump in `wrap` values. Tiles that
+        // cover a similar area of the screen can have different wrap values. This
+        // calculates this difference in wrap values so that we can match tiles
+        // across frames where the longitude gets wrapped.
+        const prevLng = this._prevLng === undefined ? lng : this._prevLng;
+        const wrapDelta = Math.round((lng - prevLng) / 360);
```

Comment:
Would this be clearer?
```js
const lngDifference = lng - prevLng;
const worldDifference = lngDifference / 360;
const wrapDelta = Math.round(worldDifference);
```

Reply:
👍 `worldDifference` definitely helps, but it's still not obvious to me why `wrapDelta` is 1 for `0.5 <= worldDifference < 1.5` rather than for, say, `1 <= worldDifference < 2`.

Reply:
Btw, I recently learned that in JS, `Math.round(-1.5)` is -1, not -2 as you might expect from other languages.

Reply:
`wrapDelta` is for `0.5 <= worldDifference < 1.5` because it is matching with the nearest frame and the sign of the distance to the nearest frame doesn't matter. If the map goes from `lng: 270` to `lng: 10` then the former `z/x/y/1` tile is closer to the current area of the `z/x/y/0` tile than the current `z/x/y/1` tile.

Any ideas on how I could clarify that well with a code or comment change?

Reply:
Now that I'm seeing the expanded comment you added, together with these var names, I think it's pretty clear  🙇